brave/browser-laptop
# Updates wizard footer text

Submitter Checklist:


Resolves #11487

Auditors:

Test Plan:
- open payment wizard
- check if text is the same as defined in the issue

Reviewer Checklist:

Tests
